meta-refkit-core/recipes-security/swtpm2/swtpm2_git.bb

```diff
@@ -0,0 +1,61 @@
+SUMMARY = "SWTPM2.0 - Software TPM2.0 Emulator"
+LICENSE = "BSD-3-Clause"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=fe8092c832b71ef20dfe4c6d3decb3a8"
+SECTION = "apps"
+
+DEPENDS = "libtasn1 fuse expect socat glib-2.0 libtpms2"
+
```

Comment:
how about adding ```PROVIDES``` for these recipes? We'd avoid making changes to '2' and back.

Reply:
You mean, override swtpm*.bb with swtpm2*.bb? I didn't want to do that because I want to keep the choice open between using one or the other.

Reply:
Yes. My thinking was to make the selection easier for the end-user. Now modifications are needed in *many* places.